datafusion/expr/src/function.rs

```diff
@@ -157,6 +157,7 @@ pub fn return_type(
             utf8_to_int_type(&input_expr_types[0], "octet_length")
         }
         BuiltinScalarFunction::Random => Ok(DataType::Float64),
+        BuiltinScalarFunction::Uuid => Ok(DataType::Utf8),
```

Comment:
I wonder if we want to try and return bytes (`DataType::Binary`) instead of strings (and allow casting from bytes to string for anyone who wants strings) 🤔

Reply:
do you think binary is a better type than string? in this case the `-` delimited hex is a standardized form however using binary people might confuse with 16 byte array...?